R/package.r

```diff
@@ -74,15 +74,19 @@ read_desc <- function(path = ".") {
 
 # Metadata ----------------------------------------------------------------
 
-read_meta <- function(path) {
-  path <- path_first_existing(
+pkgdown_config_path <- function(path) {
```

Comment:
not in scope for this PR but with such a function one could easily have a helper `file.edit(pkgdown_config_path("."))`